app/Models/MapLayer.php

```diff
@@ -87,4 +93,52 @@ public function typeName(): string
         }
         return 'overlay_shown';
     }
+
+    /**
+     * Functions for the datagrid2
+     * @return string
+     */
+    public function deleteName(): string
```

Comment:
`displayName()` is not needed  when you simply return the name, see `resources/views/layouts/datagrid/actions.blade.php`